eagles-food-backend/Services/UserRepository/UserService.cs

```diff
@@ -152,9 +153,9 @@ public UserService(LunchDbContext db_context, IMapper mapper, AuthenticationClas
                     res.Remove("LunchSenders");
                     res.Remove("Withdrawals");
                     res.Remove("IsDeleted");
-                    
+                    res.Remove("Org");
                     res.Add("access_token", token);
-
+                    res.Add("organization_name", user_login?.Org?.Name);
```

Comment:
docs are gonna have to be updated for this

Reply:
I'll update the docs.